scripts/items/pelican_ring.lua

```diff
@@ -0,0 +1,49 @@
+-----------------------------------
+-- ID: 15554
+-- Item: Pelican Ring
+-- Fishing Skillup Rate increase
+-----------------------------------
+-- Duration: 20:00 min
+-----------------------------------
+local itemObject = {}
+
+itemObject.onItemCheck = function(target)
+    if target:getMod(xi.mod.PELICAN_RING_EFFECT) >= 2 then -- Can stack effects of 2 rings
+        return xi.msg.basic.ITEM_UNABLE_TO_USE_2
+    end
+
+    return 0
+end
+
+itemObject.onItemUse = function(target, user, item)
+    local effect   = xi.effect.ENCHANTMENT
+    local power    = 0
+    local tick     = 0
+    local duration = 1200
+    local subtype  = xi.item.PELICAN_RING
+    local subpower = 0
+    local tier     = 0
+    local flag     = xi.effectFlag.ON_ZONE
+
+    -- Allow for duplicate enchantment effects, max 2
+    if target:getMod(effect, subtype) < 2 then
+        target:addStatusEffectEx(effect, effect, power, tick, duration, subtype, subpower, tier, flag)
+    end
+end
+
+itemObject.onEffectGain = function(target, effect)
+    if
+        target:getMod(xi.mod.PELICAN_RING_EFFECT) < 2
+    then
```

Comment:
`if target:getMod(xi.mod.PELICAN_RING_EFFECT) < 2 then`

Reply:
I'm okay to change that if you want, but the style guide on the wiki says to avoid single-line conditions:
![image](https://github.com/LandSandBoat/server/assets/105614059/30f30ad9-4388-4dc0-9241-09ee9f0e1857)

Good either way, just want to confirm!

Reply:
```lua
    if target:getMod(xi.mod.PELICAN_RING_EFFECT) < 2 then -- This is a single condition
        target:addMod(xi.mod.PELICAN_RING_EFFECT, 1)
    end
```

Is what Xaver was getting at.

```lua
if singleCondition then
   ....
end

if
    conditionA and
    condition
then
    ...
end
```

Reply:
Just realized I totally misunderstood what you were getting at here -- sorry about that!  I'll adjust this one.